platform/feature-api/api/src/pod-api/tests/uri.test.ts

```diff
@@ -0,0 +1,20 @@
+import { createUUID, isPolypodUri, PolyUri } from "../uri";
+
+describe("Test functions", () => {
+    it("generates and UUID in the required format", () => {
+        expect(createUUID()).toMatch(/^\w{8}-\w{4}-4\w{3}-\w{4}-\w{12}$/);
+    });
+    it("generates different UUIDs", () => {
+        let lastUUID = createUUID();
+        for (let i = 0; i < 100; i++) {
+            const thisUUID = createUUID();
+            expect(thisUUID).not.toEqual(lastUUID);
+            lastUUID = thisUUID;
+        }
+    });
```

Comment:
how about having separate `describe` for each function tested?
e.g.  
```
describe("createUUID()", () => {
    it("should ...", () => {
    });

    it("should ...", () => {
    });

...
});

describe("isPolypodUri()", () => {
    it("should ...", () => {
    });

    it("should ...", () => {
    });
...
});
```

this way it's easier to catch what is tested or not
(also counts as a good practice 😇 )